core/src/main/kotlin/riven/core/service/integration/NangoWebhookService.kt

```
@@ -0,0 +1,268 @@
+
+    /**
+     * Parses a UUID from a tag string value. Logs error and returns null on any parse failure.
+     */
+    private fun parseUuidTag(fieldName: String, value: String?): UUID? {
+        if (value == null) {
+            logger.error { "Auth webhook missing required tag field: $fieldName" }
+            return null
+        }
+        return try {
+            UUID.fromString(value)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)